src/components/backend-ai-resource-broker.ts

```diff
@@ -601,7 +603,51 @@ export default class BackendAiResourceBroker extends BackendAIPage {
       }
 
       this.total_slot = total_slot;
-      this.total_resource_group_slot = total_resource_group_slot;
+      if (!globalThis.backendaiclient._config.hideAgents) {
+        const status = 'ALIVE';
+        const limit = 20;
+        const offset = 0;
+        const timeout = 10 * 1000;
+        const fields = ['id', 'status', 'available_slots', 'occupied_slots', 'scaling_group'];
+        const agentSummaryList = await globalThis.backendaiclient.agentSummary.list(status, fields, limit, offset, timeout);
+        this.total_resource_group_slot = agentSummaryList.agent_summary_list.items
+          .filter((agent) => agent.scaling_group == this.scaling_group).map((agent) => {
+            const availableSlots = JSON.parse(agent.available_slots)
+            if ('cpu' in availableSlots) {
+              availableSlots['cpu'] = parseInt(availableSlots['cpu'] ?? 0);
+            } else {
+              availableSlots['cpu'] = 0;
+            }
+            if ('mem' in availableSlots) {
+              availableSlots['mem'] = parseFloat(globalThis.backendaiclient.utils.changeBinaryUnit(availableSlots['mem'], 'g'));
+            } else {
+              availableSlots['mem'] = 0;
+            }
+            if ('cuda.device' in availableSlots) {
+              availableSlots['cuda_device'] = parseInt(availableSlots['cuda.device'])
+            } else {
+              availableSlots['cuda_device'] = 0;
+            }
+            if ('cuda.shares' in availableSlots) {
+              availableSlots['cuda_shares'] = parseInt(availableSlots['cuda.shares'])
+            } else {
+              availableSlots['cuda_shares'] = 0;
+            }
+            return availableSlots;
+          })
+          .reduce((acc, curr) => {
+            Object.keys(curr).forEach((key) => {
+              acc[key] += curr[key];
+            });
+          return acc;
```

Comment:
how about giving default value just in case?

Reply:
> When I tested, the progress bar doesn't work properly. And please fix the lint errors 😉
> 
> * account A
>   <img alt="image" width="1194" src="https://user-images.githubusercontent.com/28584164/219539749-4bf386bd-93a4-41ff-83e0-99c7709d34b0.png">
> * account B
>   <img alt="image" width="1211" src="https://user-images.githubusercontent.com/28584164/219539936-5008c8fe-472c-4460-b2f3-efffb0e25a56.png">
> 
> If I understand correctly, the top gauge of the CPU should be 2/4.

You mean the account B is wrong? I don't think so. Because account B didn't create any session at all so it should display 0 / 4 core and etc. Please check the test condition again.

Reply:
okay. I misunderstood.